tests/cli/test_pipeline_steps.py

```diff
@@ -22,9 +28,21 @@ class TestComponent:
     prefix: str = PREFIX
 
 
-test_component_1 = TestComponent("example1", "example1")
-test_component_2 = TestComponent("example2", "example2")
-test_component_3 = TestComponent("example3", "example3")
+class TestComponentFactory(ModelFactory[PipelineComponent]):
+    to = ToSection()
+    from_ = FromSection()
+    enrich = False
+    validate = False
+    handlers = ComponentHandlers(None, MagicMock(), MagicMock())
+
+
+test_component_1 = TestComponentFactory.build(False)
+test_component_2 = TestComponentFactory.build(False)
+test_component_3 = TestComponentFactory.build(False)
```

Comment:
what does the `False` mean?

Reply:
Just tell the factory not to run the pydantic validation

Reply:
otherwise it would trip up on the `MagicMock`, right?

Reply:
yes!